src/components/ImportForm/utils/__tests__/submit-utils.spec.ts

```diff
@@ -73,6 +80,7 @@ describe('Submit Utils: createResources', () => {
     );
     expect(createApplicationMock).toHaveBeenCalledTimes(0);
     expect(createComponentMock).toHaveBeenCalledTimes(2);
+    expect(createIntegrationTestMock).toHaveBeenCalledTimes(1);
```

Comment:
I'm wondering why is it called if createApplicationMock was not called?

Reply:
...I guess createResources is after the application creation?

Reply:
`createResources` does create the Application, but only if it's not already created. I need to verify what happens if I add a Component to an existing Application. This might need a small change to only create the IntegrationTestScenario if not already there.

Reply:
Yup, that was a bug that manifested itself when adding a component to an existing application. Fixed it by only executing `createIntegrationTest` when an application is created. I also added a dry-run call so if there are any issues `createResources` fails early without creating partial resources.